cajita/src/Cajita_Halo.hpp

```diff
@@ -519,11 +519,11 @@ class Halo
     // alignment boundaries.
     template <class ArrayView>
     KOKKOS_INLINE_FUNCTION void
-    packElement( const Kokkos::View<char *, memory_space> &buffer,
-                 const Kokkos::View<int *[6], memory_space> &steering,
-                 const int element_idx, const ArrayView &array_view ) const
+    packElement( const Kokkos::View<char*, memory_space>& buffer,
+                 const Kokkos::View<int* [6], memory_space>& steering,
```

Comment:
Too bad these view data types get spaces in them.

Reply:
What would you prefer to see? I can try twiddle the options

Reply:
it would be nice if there was no space in Kokkos view data types but that might be hard.

Reply:
So ` Kokkos::View<int*[6],memory_space>&` but only for kokkos views? Yea, that sounds hard..